### PR TITLE
tomb: init at version 2.2

### PR DESCRIPTION
**Pull request by ethanpailes**
###### Things done
- [x] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Tomb is a wrapper script for some common encryption tools for
GNU/Linux.

### CODE REVIEW

**Comment by joachifm** [comment]
Hm, committing a patch is a bit unconventional. Is it possible you did `git add` instead of `git am` here?

File: copy [hunk 1]
```diff
@@ -0,0 +1,67 @@
+commit 3f6c80ad63bba52b1450651c298eb483c4fe2a15
```
Comment (joachifm) [inline copy]: Hm, committing a patch is a bit unconventional. Is it possible you did `git add` instead of `git am` here?

Comment (ethanpailes) [inline copy]: Oh, definitely. Let me fix this.

**Comment by joachifm** [comment]
Please squash when you're ready to merge.

**Comment by ethanpailes** [comment]
I can't figure out how to do that. I've tried `git rebase -i master` but it just results in a mess.

**Comment by ethanpailes** [comment]
I'm going to try this with a clean branch in a bit.

**Comment by joachifm** [comment]
You could try cherry-picking onto master

**Pull request closed without merging**